src/components/DeploymentsCount.vue

```diff
@@ -1,29 +1,26 @@
 <template>
-  <div ref="element">
-    <template v-if="visible && deploymentsCount">
-      {{ deploymentsCount }}
-    </template>
-  </div>
+  <template v-if="deploymentsCount">
+    <span class="deployments-count">{{ deploymentsCount }} {{ toPluralString(localization.info.deployment, deploymentsCount) }}</span>
+  </template>
+  <template v-else>
+    <span class="deployments-count--none">{{ localization.info.none }}</span>
+  </template>
```

Comment:
I'm not 100% sure since this is a v-if v-else but I think you'll need to bind attrs manually since the root element can change.

Reply:
Tested it out, the attrs do actually bind to either element here 👍

Reply:
Interesting, TIL. Thanks for double checking!